OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -29,14 +29,25 @@ public CaptureProperties(ScriptContext context, Actor self)
 		}
 
 		[Desc("Captures the target actor.")]
-		public void Capture(Actor target)
+		public bool Capture(Actor target)
```

Comment:
This change is not necessary now that you introduced the `CanCapture` method, which covers the `return false` case here. We cannot guarantee that the `CaptureActor` activity succeeds, so the `return true` case is unreliable.

Reply:
There might be a use case where one might want to check if the capture would fail. For example searching for an actor that could be captured, and send the actor to that location without capturing.

Capturing adds also ads an activity, which might not be needed.

Reply:
> Also since the request might fail it is necessary to return if it succeeded or not. How else would we know?

The problem is that the message does not tell the user if it worked or not, just if the activity is queued. Reporting a wrong result is Imho worse in this case, especially if we already have an extra method to cover if capturing is possible.

Reply:
> > Also since the request might fail it is necessary to return if it succeeded or not. How else would we know?
> 
> The problem is that the message does not tell the user if it worked or not, just if the activity is queued. Reporting a wrong result is Imho worse in this case, especially if we already have an extra method to cover if capturing is possible.

How is it a good practice to have a method being called that potentially might do nothing? The user has no way of knowing whether the action requested was queued. Hence the return value. Where is it reporting a wrong result? 

As i explained I do propose the have an extra method for other use cases not to solve the same issue.
User might want to query if an actor is capturable and send 10 rifle infantry there, and 60 seconds later if the actor is still capturable then send the engineer. Such thing would require additional calls to implement without checking the ability.

Reply:
1. When the return value is correct, it is redundant to the return value of the new method.
2. This might return `true` even if no capturing ever happens. The method just queues a capture activity which can fail itself: https://github.com/OpenRA/OpenRA/blob/c52913716c528f2ade35f9db5a7b62e724d7988c/OpenRA.Mods.Common/Activities/CaptureActor.cs#L52-L58
(Note that although this is the same conditional, it runs later when the capturing actor approaches the target, and the target might have changed between queueing and running of the activity.)